### PR TITLE
fixed download links

### PR DESCRIPTION
**Pull request by tindrew**
## Description
Enter short description

### CODE REVIEW

File: lib/platformDownloadLink.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 export const getPlatformDownloadLink = (platform: string): string => {
   const downloadLinks: { [key: string]: string } = {
-    windows: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.1/Redot_v4.3-rc.2_win64.exe.zip",
-    linux: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.1/Redot_v4.3-rc.2_linux.x86_64.zip",
-    android: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.1/Redot_v4.3-rc.2_android_editor.apk",
-    mac: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.1/Redot_v4.3-rc.2_macos.zip",
+    windows: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.2/Redot_v4.3-rc.2_windows_arm64.exe.zip",
```
Comment (Spartan322) [inline lib/platformDownloadLink.ts]: Nobody uses windows arm...

**Review by charlottewiltshire0 — CHANGES_REQUESTED** [review]
Correct the link to the 64-bit Windows version from the ARM version

**Comment by tindrew** [comment]
Should be fixed now.

**Review by mcdubhghlas — APPROVED** [review]
lol arm on windows...

File: lib/platformDownloadLink.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
   const downloadLinks: { [key: string]: string } = {
-    windows: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.1/Redot_v4.3-rc.2_win64.exe.zip",
-    linux: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.1/Redot_v4.3-rc.2_linux.x86_64.zip",
-    android: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.1/Redot_v4.3-rc.2_android_editor.apk",
-    mac: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.1/Redot_v4.3-rc.2_macos.zip",
+    windows: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.2/Redot_v4.3-rc.2_win64.exe.zip",
+    linux: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.2/Redot_v4.3-rc.2_linux.x86_64.zip",
+    android: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.2/Redot_v4.3-rc.2_android_editor.apk",
+    mac: "https://github.com/Redot-Engine/redot-engine/releases/download/redot-4.3-rc.2/Redot_v4.3-rc.2_macos.zip",
+
```
Comment (Spartan322) [inline lib/platformDownloadLink.ts]:
```suggestion
```

**Review by charlottewiltshire0 — APPROVED** [review]

**Pull request merged**